text/0000-config-transaction-library.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Create a standalone config transaction library that generates channel config creation and update envelopes
+nav_order: 3
+---
+
+- Feature Name: Create a standalone config transaction library that generates channel config creation and update envelopes
+- Start Date: 2020-03-20
+- RFC PR: (leave this empty)
+- Fabric Component: fabric, fabric-config
+- Fabric Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This feature will implement a consumable standalone library that supports generating configuration envelopes for 
+operations including application and system channel creation, channel configuration updates, and attaching 
+endorsing signatures to generated envelopes.
+
+# Motivation
+[motivation]: #motivation
+
+The motivation for this work is to provide a consistent API that is actually intended for consumption by external
+users for any operations involving channel config transactions. Currently `configtxgen` is available and frequently
```

Comment:
Consumption by 'external users' makes it sound like end users could use it. I like how you discussed in the Epic the example of somebody using the library to build a CLI... saying something similar here would help people understand the intended usage of the library.